mqtt_io/__main__.py

```diff
@@ -75,9 +77,12 @@ def main() -> None:
         sentry_sdk.set_context("config", redact_config(config))
         if issue_id is not None:
             sentry_sdk.set_tag("issue_id", issue_id)
-
-    mqtt_gpio = MqttIo(config)
-    mqtt_gpio.run()
+    try:
+        mqtt_gpio = MqttIo(config)
+        mqtt_gpio.run()
+    except BaseException:
```

Comment:
What's the reason for using `BaseException` here instead of `Exception`? I'm a little concerned that it would log "MqttIo crashed!" if you hit it with a keyboard interrupt at the right time.

Reply:
Oh, I tested something and forgot to change that back.
I fixed that now.